WordPress-iOS-Shared/Core/WPStyleGuide.m

```diff
@@ -139,111 +140,120 @@ + (UIFont *)tableviewSectionHeaderFont
     return [WPFontManager openSansBoldFontOfSize:12.0];
 }
 
+
 #pragma mark - Colors
+// https://wordpress.com/design-handbook/colors/
 
-+ (UIColor *)wordPressBlue
++ (UIColor *)colorWithR:(NSInteger)red G:(NSInteger)green B:(NSInteger)blue alpha:(CGFloat)alpha
```

Comment:
> I think most objective c developers are used to the division by now 

Maybe, but you're making an assumption. Even if people are used to it that's no reason we shouldn't try to make improvements :) 

>Why not make the naming something a little more natural and closer to the original method so that way it reads more natural and muscle memory kicks in.

The goal here was to type less. :)

> I should add, when i see the colorWithR: I still pause for a second to think and translate R in my head to Red whereas when I see colorWithRed: or colorWithRedValue: i don't really stop to think about it as it reads more naturally and doesn't need any translation.

Its more natural because that's what you're used to, of course. :)

Maybe its just a matter of individual preference.  I think of colors as RGB values, so "R" "G" "B" as args make sense to me.